ui/jobs/buff_tracker.ts

```diff
@@ -320,9 +335,12 @@ export class BuffTracker {
         // On each embolden stack changes,
         // there will be a gain effect log with a wrong duration (always 20).
         // So using stack to identify the first log.
+        cooldownAbility: kAbility.Embolden,
         gainEffect: EffectId.Embolden,
         loseEffect: EffectId.Embolden,
-        durationSeconds: 20,
+        useEffectDuration: true,
+        durationSeconds: 20 + 1, // for self tracing use, not actual time
```

Comment:
Do you mean "tracking" instead of "tracing"?  Even so, I'm not sure I understand the `+1` here, would you mind explaining more?

Reply:
Oh, sorry for typo!
This +1 is for RDM himself. Ability log line is earlier than effect log line.
Embolden cooldown first, than Embolden(self) active. So if without +1, user who set minimum time above 100 will see two Embolden icon (one actived for self and one cooldowning for party) at the last second.

Reply:
Yeah, I think an array might be better then because it would treat them both as the same buff.

Reply:
Changed to array. Embolden(self) now will be considered as the same raidbuff.